### PR TITLE
Vlad

### PR DESCRIPTION
**Pull request by Volodymyr0077**

### CODE REVIEW

**Comment by Volodymyr0077** [comment]
Pls review my code

**Comment by romantrust** [comment]
10/4

**Comment by romantrust** [comment]
10/4

**Comment by hafizeakyildiz** [comment]
test

File: src/main/java/TC_Vlad.java [hunk 1]
```diff
@@ -1,2 +1,19 @@
...
+        for (int i=2;i<f.length;i++){
+            f[i]=f[i-1]+f[i-2];
+        }
+        for (int e:f){
+            System.out.print(e+" ");
+        }
+        System.out.println();
+        String abc="abcskdnki aksjdjd ojda;sjdi jsapdj poaisdj";
+        String []arr=abc.split( "");
+        System.out.println(Arrays.toString(arr));
```
Comment (hafizeakyildiz) [inline src/main/java/TC_Vlad.java:17]: nice code !!

File: src/main/java/WonderWoman.java [hunk 2]
```diff
@@ -1,2 +1,9 @@
 public class WonderWoman {
+    public void existing(){
+        System.out.println("Wonder Woman is existing in this world ");
+    }
+    public static void main(String[] args) {
+        WonderWoman wonderWoman=new WonderWoman();
+        wonderWoman.existing();
+    }
```
Comment (hafizeakyildiz) [inline src/main/java/WonderWoman.java:8]: does not exist .

**Comment by romantrust** [comment]
10/4

**Review by muratkkk — APPROVED** [review]
murat

**Pull request merged**